src/lib/components/PageSection.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  export let title: string;
+  export let testId: string | undefined = undefined;
+</script>
+
+<div class="container" data-tid={testId}>
+  <div class="title-wrapper">
+    <h3 class="title">{title}</h3>
```

Comment:
Class `.title` is not in use.

Reply:
Would it make sense to provide a slot to be more flexible (h2, h4)?

Reply:
I wanted to discuss this with Artem. But it makes sense actually.

This way there can be sections of different levels maybe. I'll change it, thanks!